Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs

```diff
@@ -164,6 +161,13 @@ private void HandleUseItemInWorld(ICommonSession session, GridCoordinates coords
             if (!playerEnt.Transform.GridPosition.InRange(used.Transform.GridPosition, INTERACTION_RANGE))
                 return;
 
+            var activateMsg = new ActivateInWorldMessage(playerEnt, used);
+            RaiseEvent(activateMsg);
```

Comment:
Events are raised in a deferred manner, so you'll never be able to catch this in time.

Reply:
Yeh, I forgot the other PR https://github.com/space-wizards/RobustToolbox/pull/794.